### ISSUE TITLE
random error when restoring save file

### ISSUE THREAD

**Issue by Umbrasong** [hetima/DSP_LSTM#38]
![image](https://github.com/user-attachments/assets/0bd47e63-d3e6-4a80-a879-f1fd7d5514d3)

**Comment by Umbrasong** [comment]
for ease of copy/paste/search:
```ini
An error has occurred! Game version 0.10.30.23430
62 Mods used: [ModFixerOne1.3.2] [Reusable Metadata1.0.7] [ErrorAnalyzer1.2.2] [AchievementsEnabler1.4.0] [DSPCompass0.0.6] [DSPMultiplyStackSize1.0.13] [DSP Mod Save1.2.1] [DSPStarMapMemo0.0.6] [NebulaMultiplayerModApi2.0.0.24] [Bottleneck1.1.4] [ReorderDysonNodes0.1.0] [FastAbsorption1.0.0] [LDBTool3.0.1] [DSP Common API1.6.5.0] [MoreMegaStructure1.5.6] [DSP_Battle3.1.3] [LSTM0.9.1] [PlanetFinder1.1.5] [SnapBeltHeight1.1.0] [SplitterOverBelt1.1.7] [IlLine1.0.0] [DSPOptimizations1.1.17] [BulletTime1.5.0] [ReorderTechQueue1.2.0] [SphereEditorTools2.2.3] [CloseError1.0.0] [PlanetMiner3.0.9] [UncoverFoundationedOceans0.0.3] [Common API Nebula Compatibility1.6.5.0] [Galactic Scale 2 Plug-In2.15.2] [Galactic Scale 2 Nebula Compatibility Plug-In1.0.0.0] [Foundation Painter0.3.2] [AutoShuttleSetup1.8.0] [AutoStackedProduct1.0] [BuildBarTool1.0.1] [DeliverySlotsTweaks1.5.6] [BuildBarTool.CheckPlugins1.0.1] [Rebind Build Bar1.0.4] [BuildBarTool_RebindBuildBarCompat0.1.0] [MMSBottleneckCompat1.0] [RecyclableFuelRod2.0] [OverflowTank1.1] [DSP Save Game Sorter1.0.3] [DSP Show Grid1.0.0] [DSPMutableGridAltitudeRange0.0.1] [More Graphics Options1.0.0] [Deque1.0.1] [Blueprint Tweaks1.6.4] [Blueprint Tweaks Installation Checker1.6.4] [Giga Stations Nebula Compatibility2.3.7] [GigaStationsUpdated2.3.7] [Dustbin1.3.3] [Dyson Sphere Save Plug-In1.2.6.0] [AutoStationConfig1.4.0] [Trash1.0] [SphereOpt0.9.1] [LaserClearing1.0.5] [MassRecipePaste1.1.1] [MassRecipePasteCAPIcompat1.0.0] [RailgunsRetargetMini1.3.2] [Planet Vein Utilization1.0.6] [Upgrade_Amount0.0.2] 
MissingMethodException: void SignData.Reset(UnityEngine.Vector3,single,single)
LSTMMod.StationSignRenderer.Draw (PlanetFactory currentFactory);(IL_0029)
LSTMMod.LSTM+Patch.Mod_entitySignPool_Postfix (EntitySignRenderer __instance);(IL_000A)
EntitySignRenderer.Draw ();(IL_0099)
FactoryModel.OnCameraPostRender (UnityEngine.Camera cam);(IL_00D9)
MainCamera.OnPostRender ();(IL_0007)
[== Mod patches on the stack ==]
void LSTMMod.LSTM+Patch::Mod_entitySignPool_Postfix(EntitySignRenderer __instance); Draw(Postfix)
```

**Comment by hetima** [comment]
fixed in 0.9.2
thanks
